projects/plugins/jetpack/extensions/shared/memberships-modal.js

```diff
@@ -0,0 +1,291 @@
+// phpcs:disable no-undef
+
+/*
+ * Thickbox 3.1 - One Box To Rule Them All. jQuery was removed by Automattic. VErsion was heavily modified and features removed.
+ * By Cody Lindley (http://www.codylindley.com)
+ * Copyright (c) 2007 cody lindley
+ * Licensed under the MIT License: http://www.opensource.org/licenses/mit-license.php
+ */
+
+/*!!!!!!!!!!!!!!!!! edit below this line at your own risk !!!!!!!!!!!!!!!!!!!!!!!*/
+
+let TB_WIDTH = 0;
+let TB_HEIGHT = 0;
+const a8c_tb_pathToImage = 'https://s0.wp.com/i/loading/dark-200.gif';
```

Comment:
We could just not have this, or make it 
![Zrzut ekranu 2023-01-19 o 18 52 39](https://user-images.githubusercontent.com/3775068/213523083-6095bbfa-2b77-48a1-a2ef-2b7e485ae030.png)
better than this: